runtime/README.md

```diff
@@ -711,6 +711,84 @@ Command Code: `0x434B_4558` ("CKEX")
+| preamble\_owner\_LMS\_sig     | u32[1344]    | Owner LMOTS-SHA192-W4 signature                                             |
+| metadata\_vendor\_ecc384\_sig | u32[24]      | Metadata Vendor ECC384 signature                                            |
+| metadata\_vendor\_LMS\_sig    | u32[1344]    | Metadata Vendor LMOTS-SHA192-W4 signature                                   |
+| metadata\_owner\_ecc384\_sig  | u32[24]      | Metadata Owner ECC384 signature                                             |
+| metadata\_owner\_LMS\_sig     | u32[1344]    | Metadata Owner LMOTS-SHA192-W4 signature                                    |
+| metadata\_header\_revision    | u32          | Revision of the metadata header                                             |
+| metadata\_header\_reserved    | u32[3]       | Reserved                                                                    |
+| metadata\_entry\_entry\_count | u32          | number of metadata entries                                                  |
+| metadata\_entries             | MetaData[16] | The max number of metadata is 16 but less can be used                       |
+
+
+*Table: `AUTH\_MANIFEST\_FLAGS` input flags*
+
+| **Name**                  | **Value** |
+|---------------------------|-----------|
+| VENDOR_SIGNATURE_REQUIRED | 1 << 0    |
+
+*Table: `AUTH\_MANIFEST\_METADATA\_ENTRY` digest entries*
+
+| **Name**      | **Type** | **Description**        |
+|---------------|----------|------------------------|
+| digest        | u32[48]  | Digest of the metadata |
+| image\_source | u32      | Image source           |
+
+*Table: `SET\_AUTH\_MANIFEST` output arguments*
+
+| **Name**      | **Type** | **Description**
+| --------      | -------- | ---------------
+| chksum        | u32      | Checksum over other output arguments, computed by Caliptra. Little endian.
+| fips\_status  | u32      | Indicates if the command is FIPS approved or an error.
+
+
+### AUTHORIZE\_AND\_STASH
+
+Command Code: `0x4154_5348` ("ATSH")
+
+*Table: `AUTHORIZE\_AND\_STASH` input arguments*
+| **Name**    | **Type** | **Description**                                                                     |
+| chksum      | u32      | Checksum over other input arguments, computed by the caller. Little endian.         |
+| metadata    | u8[4]    | 4-byte measurement identifier.                                                      |
+| measurement | u8[48]   | Digest of measured                                                                  |
+| context     | u8[48]   | Context field for `svn`; e.g., a hash of the public key that authenticated the SVN. |
+| svn         | u32      | SVN                                                                                 |
+| flags       | u32      | Flags                                                                               |
+| source      | u32      | Source number                                                                       |
```

Comment:
Source is an enum right? Think we should be more clear about that.

Reply:
Updated.